app/scripts/inpage.js

```diff
@@ -55,9 +58,21 @@ if (shouldInjectProvider()) {
     target: CONTENT_SCRIPT,
   });
 
-  initializeProvider({
+  const MetaMaskEIP6963ProviderInfo = {
+    walletId: 'io.metamask',
```

Comment:
This should be based on the build type. `io.metamask` for stable, `io.metamask.flask` for Flask etc.